edi_oca/models/edi_exchange_consumer_mixin.py

```diff
@@ -115,7 +116,11 @@ def _get_eval_context(self):
     def get_view(self, view_id=None, view_type="form", **options):
         res = super().get_view(view_id, view_type, **options)
         if view_type == "form":
+            View = self.env["ir.ui.view"]
+            if view_id and res.get("base_model", self._name) != self._name:
+                View = View.with_context(base_model_name=res["base_model"])
             doc = etree.XML(res["arch"])
+            all_models = res["models"].copy()
```

Comment:
this backup is needed because...?

Reply:
Because I forgot to review if new models are added on the new process_view. Now it should work properly

Reply:
a comment would help our future selves reminding why :wink: